test/e2e/framework/framework.go

```diff
@@ -18,20 +18,25 @@ package framework
 
 import (
 	"bytes"
+	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"reflect"
 	"strings"
 	"sync"
 	"time"
 
+	"k8s.io/client-go/pkg/client/clientset_generated/release_1_4"
+	clientreporestclient "k8s.io/client-go/pkg/client/restclient"
```

Comment:
And if this could be "k8s.io/client-go/restclient" or just "k8s.io/client-go/rest"